### PR TITLE
Issue 12 ✨ feature in send asset show the asset balance of the sender

### PR DESCRIPTION
**Pull request by treasurechic**
# ℹ Overview

### CODE REVIEW

**Review by atrefonas — CHANGES_REQUESTED** [review]
Please show the balance when I fill in these two parts of the form:

image

File: pages/send-asset.js [hunk 1]
```diff
@@ -74,13 +83,46 @@ export function SendAssetPage() {
...
+      // console.log(responseData)
+      if (responseData && responseData.error == false) {
+        setAssetBalance({ success: true, message: responseData.balance })
+      } else {
+        setAssetBalance({
+          success: false,
+          message:
+            responseData?.data?.message ||
+            // eslint-disable-next-line max-len
+            'An error occurred while getting your asset balance, pls ensure you enter a valid asset id',
```
Comment (atrefonas) [inline pages/send-asset.js]: Can you change this to 'please' instead of 'pls'?

Comment (treasurechic) [reply pages/send-asset.js]: Ok

File: pages/send-asset.js [hunk 2]
```diff
@@ -94,23 +94,21 @@ export function SendAssetPage() {
...
   setTimeout(() => {
-    getAssetBalance({ wallet, assetId })
+    getAssetBalance()
   }, 3000)
 
-  const getAssetBalance = async ({ wallet, assetId }) => {
+  const getAssetBalance = async () => {
+    console.log('first', wallet, assetId)
     if (wallet && assetId) {
+      console.log('sec', wallet, assetId)
```
Comment (atrefonas) [inline pages/send-asset.js]: Can you remove or comment our the console.log's?

**Review by atrefonas — CHANGES_REQUESTED** [review]

**Review by atrefonas — APPROVED** [review]

**Pull request merged**